compiler/src/main/scala/dx/compiler/ApplicationCompiler.scala

```diff
@@ -441,9 +441,9 @@ case class ApplicationCompiler(typeAliases: Map[String, Type],
         // The reorg applet requires higher permissions to organize the output directory.
         Some(DxAccess.empty.copy(project = Some(DxAccessLevel.Contribute)))
       case _ =>
-        // Scatters need network access, because they spawn subjobs that (may) use dx-docker.
-        // We end up allowing all applets to use the network
-        Some(DxAccess.empty.copy(network = Some(Vector("*"))))
+        // ExecutableKindWfFragment / ExecutableKindWfInputs /
+        // ExecutableKindWfOutputs / ExecutableKindWfCustomReorgOutputs
```

Comment:
There could be cases when file dependences require access to other projects. But since we already state that in the details of the applet
https://github.com/dnanexus/dxCompiler/blob/9c8a47f17c7ede622ef9c2dd0331c9b917b1834c/compiler/src/main/scala/dx/compiler/ApplicationCompiler.scala#L364-L398
we don't have to add access as default here.